scipy/stats/qmc.py

```diff
@@ -0,0 +1,648 @@
+"""Quasi-Monte Carlo methods.
+
+Define function to generate sample of points in the unit hypercube.
+
+"""
+
+from __future__ import division
+
+import copy
+import numpy as np
+from scipy.optimize import brute
+from scipy._lib._util import check_random_state
+
+try:
+    from scipy.optimize import basinhopping
+
+    have_basinhopping = True
+except ImportError:
```

Comment:
Why would this ever happen?

Reply:
This was here because of the minimal supported version of scipy in statsmodels. But sure would be glad to remove :)